### ISSUE TITLE
HACS compliant

### ISSUE THREAD

**Issue by theOrakle** [mikelawrence/homeassistant-custom-components#3]
Any thoughts on setting up senseme for a HACS deployment/updates?

I just bought an i6 and was hoping to use your code, but I'm afraid I will branch your code as you make updates.

**Comment by mikelawrence** [comment]
I had never heard of Home Assistant Community Store until you mentioned it. My first instinct is no because it is not part of Home Assistant but I need to think about it. The other problem is while the Senseme component works it has some problems due to the underlying library and I'm not sure I want the support issues associated with wider usage.

**Comment by theOrakle** [comment]
Totally get it.

I can take & branch your code then.

FYI, you can setup everything for HACS and just not publish the app.  This would allow for manual setup of the repo.  Just a thought.

I did the same thing for my opensprinkler integration...



If you are interested in collaborating, let me know.  I’ll have my fan in by the end of Jan and will be kicking the integration into gear in Feb.

Thanks for the heads up on the underlying lib.

Gabriel Wieskamp
gabe.wieskamp@gmail.com

**Comment by JakeHertenstein** [comment]
As a first step, I'd suggest breaking this repository into individual components. Thanks for your efforts on the senseme component! I found that this works pretty well so far.

**Comment by sdwilsh** [comment]
It looks like you did this in https://github.com/mikelawrence/senseme-hacs :)

**Comment by mikelawrence** [comment]
You are correct. I have made the SenseME component HACS compatible. I plan on breaking the rest of these components into separate repositories so that this all-in-one repository will go away.
